### PR TITLE
add: alephium

### PR DESCRIPTION
**Pull request by sven-hash**
Adding [Alephium](alephium.org) token
* Coinpaprika page: https://coinpaprika.com/coin/alph-alephium/
* API: https://api.coinpaprika.com/v1/tickers/alph-alephium

### CODE REVIEW

**Comment by sven-hash** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by sven-hash** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/duneanalytics/spellbook/blob/main/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> I have read the CLA Document and I hereby sign the CLA
> 
> You can retrigger this bot by commenting **recheck** in this Pull Request

I have read the CLA Document and I hereby sign the CLA

File: dbt_subprojects/tokens/models/prices/ethereum/prices_ethereum_tokens.sql [hunk 1]
```diff
@@ -1821,7 +1821,8 @@ FROM
     ('ryu-ryujin', 'ethereum', 'RYU ', 0xca530408c3e552b020a2300debc7bd18820fb42f, 18),
     ('biao-biaoqing', 'ethereum', 'BIAL ', 0x9fd9278f04f01c6a39a9d1c1cd79f7782c6ade08, 9),
     ('matt-matt-furie', 'ethereum', 'MATT ', 0x790814Cd782983FaB4d7B92CF155187a865d9F18, 9),
-    ('spurdo-spurdo-on-eth', 'ethereum', 'SPURDO ', 0x3927fb89f34bbee63351a6340558eebf51a19fb8, 18)
+    ('spurdo-spurdo-on-eth', 'ethereum', 'SPURDO ', 0x3927fb89f34bbee63351a6340558eebf51a19fb8, 18),
+    ('alph-alephium', 'ethereum', 'ALPH', 0x590f820444fa3638e022776752c5eef34e2f89a6, 18)
```
Comment (amadeous) [inline dbt_subprojects/tokens/models/prices/ethereum/prices_ethereum_tokens.sql:1825]: Other tokens seems to have a space after the ticker name. We maybe should have one as well?

Comment (jeff-dude) [reply dbt_subprojects/tokens/models/prices/ethereum/prices_ethereum_tokens.sql:1825]: hmm this actually seems like a missed bug, can we remove the space on those few rows instead? looks like ~10-15 lines have it

**Review by jeff-dude — APPROVED** [review]

**Comment by sven-hash** [comment]
Is there any issue @jeff-dude ?

**Comment by jeff-dude** [comment]
> Is there any issue @jeff-dude ?

not with your new token. trying to figure out why some older ones are causing issue.

**Comment by sven-hash** [comment]
> > Is there any issue @jeff-dude ?
> 
> not with your new token. trying to figure out why some older ones are causing issue.

Can you let me know where is the issue? The API is here https://api.coinpaprika.com/v1/tickers/alph-alephium

**Review by jeff-dude — APPROVED** [review]

**Comment by sven-hash** [comment]
Thanks for your help @jeff-dude

**Pull request merged**